puppet/modules/slave/manifests/rbenv.pp

```diff
@@ -0,0 +1,24 @@
+# rbenv Configuration
+# @api private
+class slave::rbenv {
+  class { 'rbenv':
+    #owner       => 'jenkins',
+    #group       => 'jenkins',
+    #install_dir => '/home/jenkins/.rbenv',
+    #require     => User['jenkins'],
```

Comment:
Commented out, as I think this is what we want to do rather than doing it for the whole system as root? The only problem is the error I get right now:

```
==> jenkins-node-el9: Notice: /Stage[main]/Slave::Rbenv/Rbenv::Build[3.1.0]/Exec[git-pull-rubybuild-3.1.0]/returns: fatal: detected dubious ownership in repository at '/home/jenkins/.rbenv/plugins/ruby-build'
==> jenkins-node-el9: Notice: /Stage[main]/Slave::Rbenv/Rbenv::Build[3.1.0]/Exec[git-pull-rubybuild-3.1.0]/returns: To add an exception for this directory, call:
==> jenkins-node-el9: Notice: /Stage[main]/Slave::Rbenv/Rbenv::Build[3.1.0]/Exec[git-pull-rubybuild-3.1.0]/returns: 
==> jenkins-node-el9: Notice: /Stage[main]/Slave::Rbenv/Rbenv::Build[3.1.0]/Exec[git-pull-rubybuild-3.1.0]/returns: 	git config --global --add safe.directory /home/jenkins/.rbenv/plugins/ruby-build
==> jenkins-node-el9: Error: 'git reset --hard HEAD && git pull' returned 128 instead of one of [0]
==> jenkins-node-el9: Error: /Stage[main]/Slave::Rbenv/Rbenv::Build[3.1.0]/Exec[git-pull-rubybuild-3.1.0]/returns: change from 'notrun' to ['0'] failed: 'git reset --hard HEAD && git pull' returned 128 instead of one of [0]
```

I think this is related to how it performs things as root and then uses chown to update owner and group.